### PR TITLE
[1321] Remove change vacancies from course show page

### PR DESCRIPTION
**Pull request by defong**
### Context

`change vacancies` link on course show page

### Changes proposed in this pull request

Remove `change vacancies` link on course show page

#### Before
image

#### After
image

### Checklist

- [x] Make sure all information from the Trello card is in here
- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally
- [x] Inform data insights team due to database changes

### CODE REVIEW

File: app/services/feature_service.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def enabled?(feature_name)
 
       segments = feature_name.to_s.split('.')
 
-      segments.reduce(Settings.features) { |config, segment| config[segment] }
+      segments.reduce(Settings.features) { |config, segment| config[segment] } == true
```
Comment (goodviber) [inline app/services/feature_service.rb:16]: Why this change?

Comment (defong) [reply app/services/feature_service.rb:16]: The method signature is

`    def enabled?(feature_name)`

So it shoud only return a true or a false given enabled`?` 

Without it any value can be returned including `nil`
